cea/demand/metamodel/nn_generator/input_matrix.py

```diff
@@ -10,14 +10,15 @@
 import pandas as pd
 import cea.globalvar
 import cea.inputlocator
+import cea.config
 from cea.demand.demand_main import properties_and_schedule
 from cea.demand.thermal_loads import initialize_inputs
 from cea.utilities.dbfreader import dbf_to_dataframe
 from cea.technologies import controllers
 from cea.utilities import epwreader
 from cea.demand.sensible_loads import calc_I_sol
 import numpy as np
-from cea.demand.metamodel.nn_generator.nn_settings import nn_delay, climatic_variables
+from cea.config.neural_network import nn_delay, climatic_variables
```

Comment:
did you create a `cea/config` directory? i'm not sure this line can even work...